csvcubed-devtools/pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "csvcubed-devtools"
 packages = [
   {include = "csvcubeddevtools"},
 ]
-version = "0.0.1"
+version = "0.1.0-rc1"
```

Comment:
I wouldn't do this. The version number is to be updated by the auto-tag-version-thingy as described in the release proceedure.

Reply:
Same for each of the other version changes you've made too.

Reply:
I didn't make any changes, I created the branch from v0.1.0-rc1. The new thing will overwrite them anyways.

Reply:
Ok, but you're merging it into main so it's confusing who sets/where the version gets set.